### PR TITLE
0.12 pickles update

### PR DESCRIPTION
**Pull request by radinamatic**
### Summary

*Ready for review & merge!*

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] Gherkin stories have been updated

### Reviewer Checklist

- PR is fully functional
- Documentation is updated

### CODE REVIEW

**Comment by indirectlylit** [comment]
minor note - recommend not removing checkboxes from the PR template

**Comment by radinamatic** [comment]
> recommend not removing checkboxes from the PR template

Noted, but even when they are not applicable at all?

**Comment by indirectlylit** [comment]
> even when they are not applicable at all?

correct - just leaving them unchecked is fine

File: integration_testing/features/super-admin/super-admin-profile-command.feature [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
+      And I browse the "KOLIBRI_HOME/performance" folder again
+    Then I see another file called "xxxxxxxx_xxxxxx_requests_performance.csv"
+      And it has the same date_time as the "xxxxxxxx_xxxxxx_performance.csv" file
+
+  Scenario: Stop interacting with server and observe profiling results 
+    Given that 1 minute has passed after running the command 
+    When I see the prompt in the Terminal again # Profiling requests is finished, and CSV files stop changing
+    Then I open CSV files in a text editor or a spreadsheet application
+      And I see that "xxxxxxxx_xxxxxx_performance.csv" file has at least 6 lines beginning with a timestamp followed by profiling results # First line is a header
+      And I see that "xxxxxxxx_xxxxxx_requests_performance.csv" file has at least one line per each Kolibri page I interacted with in the browser, and lines begin with a timestamp followed by requests profiling results
```
Comment (indirectlylit) [inline integration_testing/features/super-admin/super-admin-profile-command.feature:24]: great

File: yarn.lock [hunk 2]
```diff
@@ -5,14 +5,12 @@
 "@babel/code-frame@^7.0.0", "@babel/code-frame@^7.0.0-beta.35":
   version "7.0.0"
   resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.0.0.tgz#06e2ab19bdb535385559aabb5ba59729482800f8"
-  integrity sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==
```
Comment (indirectlylit) [inline yarn.lock]: please revert the changes to this file

**Review by indirectlylit — CHANGES_REQUESTED** [review]
changes look great!

please revert the changes to yarn.lock - you may need to update your yarn installation locally

**Comment by radinamatic** [comment]
@indirectlylit `yarn.lock` has been reverted, let's wait for the checks to pass and it can be merged!

**Review by indirectlylit — APPROVED** [review]

**Comment by indirectlylit** [comment]
skipping checks because it's just feature files

**Pull request merged**